client/cli/ok.py

```diff
@@ -109,8 +112,7 @@ def main():
             auth.authenticate(True)
 
         # Instantiating assignment
-        assign = assignment.load_config(args.config, args)
-        assign.load()
+        assign = assignment.load_assignment(args.config, args)
```

Comment:
Whoops, read this more carefully. It seems that `Assignment.load` was merged into `Assignment.post_instantiation`.

In that case, `client/cli/lock.py` and `client/cli/test.py` both still have `assign.load()` which I think should be removed.